apps/yggdrasil/priv/repo/migrations/20160303191523_create_users_roles.exs

```diff
@@ -0,0 +1,15 @@
+defmodule Yggdrasil.Repo.Migrations.CreateUsersRoles do
+  use Ecto.Migration
+
+  def change do
+    create table(:users_roles, primary_key: false) do
+      add :user_id, references(:users, on_delete: :nothing)
+      add :role_id, references(:roles, on_delete: :nothing)
+
+      timestamps
+    end
+    create index(:users_roles, [:user_id])
```

Comment:
Same here, would you need an index of `:user_id` if you have a unique constraint with `:user_id` as the first column?